nilearn/maskers/nifti_labels_masker.py

```diff
@@ -44,7 +44,7 @@ def __call__(self, imgs):
 
 
 @_utils.fill_doc
-class NiftiLabelsMasker(BaseMasker, _utils.CacheMixin):
+class NiftiLabelsMasker(BaseMasker):
```

Comment:
CacheMixin was removed (and other maskers) here because it is already part of the BaseMasker: AFAIU this should not affect behavior. 🤞🏾